dotfiles

```diff
@@ -167,6 +167,8 @@ edit_info() {
 sparse_edit() {
     local sparse_file="$1"; shift
     local temp_dir="$(mktemp -dt dotfiles.XXXXXX)"
+    mkdir --parents "$temp_dir/$(dirname "$sparse_file")"
+    echo "!$sparse_file" >> "$dotfiles_dir"/info/sparse-checkout
```

Comment:
I just realized, that sparse checkouts obviously only work locally. That is, if I add a new sparse-checkouted file on one computer and then pull this change, on the other computer it's not sparse-checkouted. Do you have any idea how to solve this? As a temporary fix, I propose to sparse-checkout `.github/` by default